docs/content/users/extend/additional-services.md

```diff
@@ -14,49 +13,53 @@ For example,
 ┌───────────────────────────┬──────────────────────────────────────────────────┐
 │ ADD-ON                    │ DESCRIPTION                                      │
 ├───────────────────────────┼──────────────────────────────────────────────────┤
-│ drud/ddev-memcached       │ Install Memcached as an extra service in DDEV*   │
+│ drud/ddev-adminer         │ Adminer service for DDEV*                        │
```

Comment:
If this were in a ``` block you wouldn't have to add it to spellcheck.

Reply:
The part you referenced here _is_ in a fenced code block, it’s the “Adminer” in the bulleted list further down that caused the kerfuffle.